src/pages/_app.tsx

```diff
@@ -2,15 +2,41 @@ import type { AppProps } from "next/app"
 // ChakraProvider import updated as recommended on https://github.com/chakra-ui/chakra-ui/issues/4975#issuecomment-1174234230
 // to reduce bundle size. Should be reverted to "@chakra-ui/react" in case on theme issues
 import { ChakraProvider } from "@chakra-ui/provider"
+import { Inter } from "next/font/google"
+import { IBM_Plex_Mono } from "next/font/google"
 
 // Chakra custom theme
 import theme from "@/@chakra-ui/theme"
 
+const inter = Inter({
```

Comment:
we could move `inter` and `mono` declarations into `@lib/fonts` to clean `_app` a bit

Reply:
Agreed.